src/public/js/rs.js

```diff
@@ -21,6 +105,94 @@ var RS = (function() {
     }, '');
   }
 
+  // Encodes using systematic BCH Reed-Solomon on Galois Field 256
```

Comment:
GF(256) only exists using GF(2) as its base field so this isn't a concern. It is just notation but the same meaning.